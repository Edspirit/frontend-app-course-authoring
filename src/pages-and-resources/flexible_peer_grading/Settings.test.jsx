import { shallow } from 'enzyme';
import FlexiblePeerGradingSettings from './Settings';

jest.mock('@edx/frontend-platform/i18n', () => ({
  ...jest.requireActual('@edx/frontend-platform/i18n'), // use actual for all non-hook parts
  injectIntl: (component) => component,
  intlShape: {},
}));
jest.mock('../app-settings-modal/AppSettingsModal', () => 'AppSettingsModal');

const props = {
  onClose: jest.fn().mockName('onClose'),
  intl: {
    formatMessage: (message) => message.defaultMessage,
  },
};

describe('FlexiblePeerGradingSettings', () => {
  it('should render', () => {
    const wrapper = shallow(<FlexiblePeerGradingSettings {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
