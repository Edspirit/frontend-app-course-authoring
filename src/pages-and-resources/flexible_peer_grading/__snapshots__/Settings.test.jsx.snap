// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlexiblePeerGradingSettings should render 1`] = `
<AppSettingsModal
  appId="flexible_peer_grading"
  enableAppHelp="Flexible peer grading for all open response assessments in the course."
  enableAppLabel="Flex Peer Grading"
  learnMoreText="Learn more about flexible peer grading"
  onClose={[MockFunction onClose]}
  title="Configure ORA Flexible Peer Grading"
/>
`;
